### PR TITLE
[SW-193] Total wallet amount with token amount fix

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

File: src/popup/router/components/AeBalance.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
   <div class="ae-balance">
     <span class="token-symbol">Æ </span>
-    <span class="token-integer">{{ `${integerPart}.` }}</span>
-    <span class="token-fractional">{{ fractionalPart }}</span>
+    <span class="token-integer">{{ `${balanceParts.integer}.` }}</span>
```
Comment (peronczyk) [inline src/popup/router/components/AeBalance.vue:4]: Integer part is already converted to string.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/router/components/AccountSwitcher.vue [hunk 2]
```diff
@@ -3,6 +3,7 @@
     class="account-switcher"
     :class="{ 'notification-above': notification }"
   >
+    <TotalWalletAmount v-if="filteredAccounts.length > 1" />
```
[the remark this answers is not in the document]
Comment (pstachel) [reply src/popup/router/components/AccountSwitcher.vue:6]: Well, it's used only in account switcher and requires 'filteredAccounts' property from AccountSwitcher, so I think it should stay there

File: src/popup/router/components/TotalWalletAmount.vue [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import { mapGetters } from 'vuex';
+import { pick } from 'lodash-es';
+import BigNumber from 'bignumber.js';
+
+export default {
+  subscriptions() {
+    return pick(this.$store.state.observables, ['balances']);
+  },
+  computed: {
+    ...mapGetters(['convertToCurrencyFormatted', 'accounts']),
```
Comment (CedrikNikita) [inline src/popup/router/components/TotalWalletAmount.vue:20]:
```suggestion
    ...mapGetters(['convertToCurrencyFormatted']),
```
`accounts` is not used in this component.

File: src/popup/router/components/TotalWalletAmount.vue [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    totalAmount() {
+      if (!this.balances) return 0;
```
Comment (CedrikNikita) [inline src/popup/router/components/TotalWalletAmount.vue:22]: https://github.com/aeternity/superhero-wallet/blob/67f25570223516b83792d9ffddc28e90daf484fa/src/store/plugins/observables.js#L66-L72

I don't think that this condition would work.

**Review by peronczyk — APPROVED** [review]

File: src/popup/router/components/AeBalance.vue [hunk 5]
```diff
@@ -12,11 +12,12 @@ export default {
     balance: { type: Number, default: 0 },
   },
   computed: {
-    integerPart() {
-      return Math.trunc(this.balance).toLocaleString();
-    },
-    fractionalPart() {
-      return (Math.trunc((this.balance % 1) * 100));
+    balanceParts() {
+      const [integer, fraction] = this.balance.toFixed(2).toString().split('.');
```
Comment (konradodo) [inline src/popup/router/components/AeBalance.vue:16]: toFixed(2) that return string

**Pull request merged**